.github/workflows/ci.yml

```diff
@@ -125,11 +115,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/alicenet-config
-      - name: install-golangci-lint
```

Comment:
Maybe we should continue using the curl version here. We used the pre-compiled version of the golangci, some weeks ago. However, that was not displaying the errors correctly. Getting the version via curl solved all problems.
Linter output on this PR:
https://github.com/MadBase/MadNet/runs/6575091389?check_suite_focus=true
Linter output on other PR:
https://github.com/MadBase/MadNet/runs/6574348993?check_suite_focus=true

Reply:
After discussion, we saw that this error may have been fixed on the latest version of the golinter Github's action.